src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -496,6 +503,7 @@ export class SearchInterface extends RootComponent implements IComponentBindings
   public componentOptionsModel: ComponentOptionsModel;
   public usageAnalytics: IAnalyticsClient;
   public historyManager: IHistoryManager;
+  public scrollController: ScrollRestorationController;
```

Comment:
It does not seem like keeping a reference here serve any use case.

The other reference we keep (eg: usageAnalytics) is because there's code out there that needs to reference/resolve those instance through the search interface.

For the scroll restoration, it does not seem to be needed.